addons/purchase/controllers/portal.py

```diff
@@ -43,7 +45,7 @@ def _prepare_portal_overview_values(self):
         })
         return values
 
-    def _render_portal(self, template, page, date_begin, date_end, sortby, filterby, domain, searchbar_filters, default_filter, url, history, page_name, key):
+    def _render_portal(self, template, page, date_begin, date_end, sortby, filterby, domain, searchbar_filters, default_filter, url, history, page_name, key, searchbar_inputs, search_domain, search, search_in):
```

Comment:
Optional parameters at least ?

+ multi line ?

Reply:
optional parameters are set in the route, this function is always called by a route which set all parameters